nilearn/maskers/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,167 @@
+"""
+Transformer for computing ROI signals of multiple 4D images
+"""
+
+import itertools
+
+from joblib import Parallel, delayed
+
+from .._utils import CacheMixin, fill_doc
+from .._utils.niimg_conversions import _iter_check_niimg
+from .nifti_labels_masker import NiftiLabelsMasker
+
+
+@fill_doc
+class MultiNiftiLabelsMasker(NiftiLabelsMasker, CacheMixin):
```

Comment:
niftilabelsmaskers is already derived from CacheMixin so I don't think we need CacheMixin here